README.md

```diff
@@ -203,107 +203,78 @@ All messages must pass the following validations in addition to specific validat
 
 ## 4.1 Casts
 
-A _Cast_ is a public message created by a user that is displayed on their profile. It can contain text and links to media, on-chain activity or other casts. Users are allowed to delete casts at any time. Casts come in several different flavors and the protocol can be extended to support many more types in the future. Each type has its own replication mechanisms which are defined below.
+A _Cast_ is a public message created by a user which contains text and can also embed media, on-chain activity or evens other casts. Casts are stored in a two-phase set CRDT[^two-phase-set] that handles conflict resolution.
 
-### 4.1.1 Short Text Casts
+A Cast can be added with a `CastAdd` message which is placed in the CRDT's **add-set**. Each cast is indexed by its hash which is guaranteed to be unique unless the casts are identical. By extension, two add messages can never conflict unless they are identical, in which case one can be discarded safely. 
```

Comment:
Annoying nit: "add-set" is bolded here but "rem-set" is not bolded in the next paragraph